OsmAnd/res/values/strings.xml

```diff
@@ -904,7 +904,7 @@
     <string name="no_recalculation_setting">No recalculation</string>
     <string name="open_settings">Open settings</string>
     <string name="plugin_disabled">Plugin off</string>
-    <string name="plugin_disabled_descr">This plugin is a separate app, you will need to remove it separately if you no longer plan to use it.\n\nThe plugin will remain on the device after removing OsmAnd.</string>
+    <string name="plugin_disabled_descr">Remove this plugin if you don't use it.\n\nIt will remain on the device after removing OsmAnd.</string>
```

Comment:
No, the hint that this is a separate app for the OS is essential.

Reply:
```suggestion
    <string name="plugin_disabled_descr">Remove this plugin if you don't use it or uninstall OsmAnd.\n\nOtherwise it remains as an app on the device.</string>
```